src/main/java/com/redhat/devtools/lsp4ij/LSPIJUtils.java

```diff
@@ -1434,4 +1437,82 @@ public static String getProjectUri(Project project) {
                 .toList();
     }
 
+    // IDE version check utilities to support branching logic based on the host IDE version
+
+    /**
+     * Determines whether or not the host IDE is at least the specified major/minor version.
+     *
+     * @param minimumMajorVersion the major version number
+     * @param minimumMinorVersion the minor version number
+     * @return true if the host IDE is at least the specified major/minor version; otherwise false
+     */
+    public static boolean isVersionAtLeast(
```

Comment:
I ported the utility methods from my plugin to check for minimum host IDE version. These are useful for when logic has to branch -- or only occur -- in a minimum IDE version when the plugin supports versions older than that as well.